server/api/expense-pool/expense/index.js

```diff
@@ -1,23 +1,23 @@
+/**
+ * POST  /api/expense-pools/:id/expenses            -> create
+ * GET   /api/expense-pools/:id/expenses            -> all expenses for a pool
+ * GET   /api/expense-pools/:id/expenses/:expenseId -> get
+ * PUT   /api/expense-pools/:id/expenses/:expenseId -> update
+ * PATCH /api/expense-pools/:id/expenses/:expenseId -> update
+ */
+
 import express from 'express';
 import controller from './expense.controller';
 
-const router = express.Router({ mergeParams: true });
```

Comment:
Because the way the authorization works in decorating the requests we don't need the params from the `expense-pool` portion of the url. The `ExpensePool` instance indicated by the `:id` from expense pool router will be a property of the `req` as `req.expensePool`.